doc/cygnus-ngsi/flume_extensions_catalogue/last_data_function.md

```diff
@@ -0,0 +1,161 @@
+# Last Data functionality.
+
+Cygnus is capable to perform an `upsert` operation on given SQL Sinks.  This Operation doesn't overrides the usual inserts on the usual way.
```

Comment:
Maybe an explicit list of which sinks supporting this feature should be included.

Reply:
I added the reference to this document on the PostgreSQLSink document. To keep a little more open this document. IMHO that's more clear. 

But sure, I can add an explicit list of the sinks. Do you want me to do it?

Reply:
> But sure, I can add an explicit list of the sinks. Do you want me to do it?

Not 100% sure, but I think is a good idea :) Note that the user could land directly in last_data_function.md so it's a good idea to know at a first glance at the top of the document which sinks support this functionality.

Reply:
Fixed in [3be5654](https://github.com/telefonicaid/fiware-cygnus/pull/1934/commits/3be56543ed106087a647eac3cb71c863780a03fb)